contrib/builder/rpm/generate.sh

```diff
@@ -57,7 +61,11 @@ for version in "${versions[@]}"; do
 		sqlite-devel # for "sqlite3.h"
 		tar # older versions of dev-tools don't have tar
 	)
-	echo "RUN yum install -y ${packages[*]}" >> "$version/Dockerfile"
+	if [ "$from" == "oraclelinux:7" ]; then
+		echo "RUN yum install --enablerepo=ol7_optional_latest -y ${packages[*]}" >> "$version/Dockerfile"
```

Comment:
Is there not a simple way to just enable this repo all the time so we could have Oracle use the same "yum install" line and not have the duplication here?